Sources/FluentKit/Properties/Relation.swift

```diff
@@ -0,0 +1,23 @@
+public protocol Relation {
+    associatedtype RelatedValue
+
+    var name: String { get }
+    var value: RelatedValue? { get set }
+
+    func load(on database: Database) -> EventLoopFuture<Void>
```

Comment:
Would it make sense to have this return `EventLoopFuture<RelatedValue>` instead of `Void`?

Reply:
@MrLotU that's what the extension `get` (just below) does using this method.

Reply:
The separation of triggering an async load and subsequent no-query access sounds more useful and efficient in practice to me.

Reply:
Ah. Duh. I shouldn't do PR reviews when I'm tired. Thanks :D